website/_layout.jade

```diff
@@ -12,4 +12,4 @@ html
     main
       != yield
 
-    script(src="js/application.js")
+    // script(src="js/application.js")
```

Comment:
I think commented out code needs an explanation about when it should be uncommented or should just be removed